Protobuild.Internal/BuildResources/GenerateProject.CSharp.xslt

```diff
@@ -69,7 +69,7 @@
   
   <xsl:template name="AllowLangVersion"
     xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
-    <xsl:if test="not($root/Input/Generation/Platform = 'MacOS') or (user:HasXamarinMacUnifiedAPI() and not(user:IsTrue($root/Input/Properties/UseLegacyMacAPI)))">
+    <xsl:if test="not($root/Input/Generation/Platform = 'MacOS') or (user:HasXamarinMacUnifiedAPI() and not(user:IsTrue($root/Input/Properties/UseLegacyMacAPI)) and not(user:IsTrue($root/Input/Properties/UseMonoMac)))">
```

Comment:
Call it UseMonoMacAPI for consistency with the other option.

I don't really like having boolean options that conflict with each other either, so this might better as `$root/Input/Properties/MacAPIVersion = 'MonoMac'`.